.gitignore

```diff
@@ -71,6 +71,7 @@ id_rsa_*
 
 # python things
 __pycache__
+.venv
```

Comment:
Could this be reverted? 

If it needs to be added, it should be done as a separate PR.